mobile_payment/migrations/0004_mobile_payment_rights.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.16 on 2023-07-07 13:38
+
+import core.fields
+import datetime
+from django.db import migrations, models
+
+from core.utils import insert_role_right_for_system
+
+
+def add_rights(apps, schema_editor):
+
+    insert_role_right_for_system(64, 208000)  # search
+    insert_role_right_for_system(64, 208001)  # create
+    insert_role_right_for_system(64, 208002)  # update
+    insert_role_right_for_system(64, 208003)  # search
+    insert_role_right_for_system(64, 208004)  # create
+    insert_role_right_for_system(64, 208005)  # search
+    insert_role_right_for_system(64, 208006)  # create
+    insert_role_right_for_system(64, 208007)  # update
+    insert_role_right_for_system(64, 208008)  # delete
```

Comment:
I'd use the `DEFAULT_CFG` constant from the module `apps.py` to have them defined only in one place.

Reply:
How do i do that then